[SHT_LLVM_BB_ADDR_MAP] Adds pretty printing of BFI and BPI for PGO Analysis Map in tools. (#82292)
llvm/llvm-project: llvm/docs/CommandGuide/llvm-objdump.rst

  When printing a PC-relative global symbol reference, print it as an offset from the leading symbol.

  When a bb-address-map section is present (i.e., the object file is built with
  ``-fbasic-block-sections=labels``), labels are retrieved from that section

>   instead. If a pgo-analysis-map present along side the bb-address-map, any

Reviewer: ```suggestion
  instead. If a pgo-analysis-map is present alongside the bb-address-map, any
```
Two nits.

       cmp eax, dword ptr <g>
       jge	<L0>

.. option:: --pretty-pgo-analysis-map

>   When using ``--symbolize-operands`` with bb-address-map and pgo-analysis-map,

Reviewer: ```suggestion
  When using :option:``--symbolize-operands`` with bb-address-map and pgo-analysis-map,
```
Also applies below - this will ensure the reference becomes a hyperlink to the actual option in the documentation.

Author: Thanks for the suggestion!

>   print analyses using same format as there analysis passes would. An example

Reviewer: ```suggestion
  print analyses using the same format as their analysis passes would. An example
```

>   of pretty format, would be printing block frequencies relative to the entry
>   block, same as BFI.

Reviewer: ```suggestion
  of pretty format would be printing block frequencies relative to the entry
  block, the same as BFI.
```

.. option:: --pretty-pgo-analysis-map

  When using :option:`--symbolize-operands` with bb-address-map and

>   pgo-analysis-map, print analyses using same the format as their analysis

Reviewer: ```suggestion
  pgo-analysis-map, print analyses using the same format as their analysis
```
